charts/legalhold/templates/deployment.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: "{{ .Release.Name }}-hold"
+  labels:
+    name: "{{ .Release.Name }}-hold"
+  checksum/secrets: {{ include (print $.Template.BasePath "/secrets.yaml") . | sha256sum }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: "{{ .Release.Name }}-hold"
+  template:
+    metadata:
+      name: "{{ .Release.Name }}-hold"
+      labels:
+        name: "{{ .Release.Name }}-hold"
+    spec:
+      restartPolicy: Always
+      containers:
+        - name: hold
+          image: quay.io/wire/hold:1.0.2
```

Comment:
Yes yes I know this could be made templateable; but lets first make it work; then come up with what can all be templated.